vmware_dvs/common/config.py

```diff
@@ -41,15 +41,28 @@
                help=_("Vsphere host name or IP.")),
     cfg.StrOpt('vsphere_password', default='',
                help=_("Vsphere password.")),
+    cfg.StrOpt('cluster_name',
+               help=_("compute_cluster_name."))
 ]
 
 dvs_opts = [
     cfg.BoolOpt('clean_on_restart',
-               default=True,
-               help=_("Run DVS cleaning procedure on agent restart.")),
+                default=True,
+                help=_("Run DVS cleaning procedure on agent restart.")),
     cfg.BoolOpt('precreate_networks',
-               default=False,
-               help=_("Precreate networks on DVS")),
+                default=False,
+                help=_("Precreate networks on DVS.")),
+    cfg.IntOpt('init_pg_ports_count',
+               default=4,
+               help=_("Initial ports size for networks on DVS.")),
+    cfg.FloatOpt('cache_pool_interval',
+                 default=0.1,
+                 help=_("The interval of task polling for "
+                        "DVS cache in seconds.")),
+    cfg.IntOpt('cache_free_ports_size',
+               default=50,
```

Comment:
too much, change for 20. It will be for each virtual network of each DVS. Really cache could be greater.